Comments/ClientApp/src/components/LoginBanner/LoginBanner.js

```diff
@@ -1,38 +1,254 @@
 // @flow
-import React, { PureComponent } from "react";
+import React, { Component, Fragment } from "react";
+import { withRouter } from "react-router-dom";
+import { DebounceInput } from "react-debounce-input";
+import queryString from "query-string";
+import Cookies from "js-cookie";
+import { UserContext } from "../../context/UserContext";
+import { load } from "../../data/loader";
+import { Input } from "@nice-digital/nds-input";
 
 type PropsType = {
 	signInURL: string,
 	registerURL: string,
 	signInButton: boolean,
 	signInText?: string,
+	match: PropTypes.object.isRequired,
+	location: PropTypes.object.isRequired,
+	allowOrganisationCodeLogin: boolean,
 }
 
-export class LoginBanner extends PureComponent<PropsType> {
+type OrganisationCode = {
+	organisationAuthorisationId: number,
+	organisationId: number,
+	organisationName: string,
+	collationCode: string,
+}
+
+type StateType = {
+	organisationCode: string,
```

Comment:
```suggestion
	userEnteredCollationCode: string,
```
Think this should be as above. I've highlighted the line below where it's missing from the constructor and where userEnteredCollationCode is set instead.